### DISCUSSION TITLE
My image has a random white section at the top. What could be causing this?

### DISCUSSION THREAD

**Discussion by denviljclarke** [cloudinary-community/next-cloudinary#177] [Q&A] [answered]
![Screenshot 2023-04-11 at 6 44 52 PM](https://user-images.githubusercontent.com/60730266/231312503-094fc796-c427-45cc-ae25-4ddeb25e635d.png)

As you can see the images have this unwanted space. What can be done to fix this?

**Comment by colbyfayock** [comment]
hey @denviljclarke - can you share with me the code of the component and the resulting Cloudinary URL?

from first glance from your screenshot, it doesn't appear that there's anything from the Cloudinary URL that would cause it, so my guess is that something is happening with the component, whether that's from CldImage or from the underlaying Next.js Image component

**Reply by denviljclarke** [reply, marked as answer]
So, upon further inspection it seems that it was **my header overlapping the image** which caused this. It seems that I somehow overlooked this & being that it's there's a similar prevalent issue with space below images in the next repo I associated it with that. in any case, thank you for the swift response @colbyfayock. 

Also, thank you for your work on this library while I have the chance 🙂

**Reply by colbyfayock** [reply]
no problem! and glad you worked it out
